test/test_va_api_fixture.h

```diff
@@ -33,6 +33,8 @@
 namespace VAAPI
 {
 
+VADisplay getDrmDisplay(int &fd);
```

Comment:
Remove declaration.  Not needed if you move SetupTestSuite/TearDownTestSuite definition to cpp file.